MailerQ/RestApi/Error.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Newtonsoft.Json;
+
+namespace MailerQ.RestApi
+{
+    /// <summary>
+    /// Using this endpoint MailerQ allows you to intercept it and forcing an error on any combination of pool/mta and domain.
+    /// </summary>
+    [JsonObject(
+    NamingStrategyType = typeof(LowercaseNamingStrategy),
+    ItemNullValueHandling = NullValueHandling.Ignore
+    )]
+    public class Error
+    {
+        /// <summary>
+        /// Numeric error code between 200 and 599 (smtp error codes).
+        /// </summary>
+        public int Code { get; set; }
```

Comment:
[Erros documentation](https://www.mailerq.com/documentation/5.8/rest-api-v1-errors) indicate that _Code_ is required, maybe it is a good idea validate it when serialize the object.
```suggestion
        [JsonProperty(Required = Required.Always)]
        public int Code { get; set; }
```